x-pack/plugins/observability/public/components/custom_threshold/components/criterion_preview_chart/criterion_preview_chart.tsx

```diff
@@ -12,7 +12,7 @@ import { i18n } from '@kbn/i18n';
 import { EuiLoadingChart, EuiText } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { sum, min as getMin, max as getMax } from 'lodash';
-import { GetLogAlertsChartPreviewDataSuccessResponsePayload } from '../../../../../common/threshold_rule/types';
+import { GetLogAlertsChartPreviewDataSuccessResponsePayload } from '../../../../../common/custom_threshold_rule/types';
 import { formatNumber } from '../../../../../common/threshold_rule/formatters/number';
```

Comment:
@fkanout I caught this one. `threshold_rule` should be renamed to `custom_threshold_rule`.

Reply:
@fkanout Looks like you fixed this issue on this [commit](https://github.com/elastic/kibana/pull/166145/commits/4492a5ac645bb76dab1c865e7092d45efa73314d).